README.md

```diff
@@ -1,8 +1,7 @@
 # 📗 Table of Contents
 
 - [📗 Table of Contents](#-table-of-contents)
-- [📖 Budget-App ](#Budget-App)
-- [Video-Presentation](#Live-Presentaton)
+- [📖 Budget-App ](#hello_rails_react)
   - [🛠 Built With ](#-built-with-)
```

Comment:
- [ ] As required by the previous code reviewer, I kindly suggest that you kindly change the name of your app from Budget App to something more suitable.